src/lib/routes/admin-api/addon.ts

```diff
@@ -288,24 +284,11 @@ Note: passing \`null\` as a value for the description property will set it to an
 
         const { limit = '50', offset = '0' } = req.query;
 
-        const normalizedLimit =
-            Number(limit) > 0 && Number(limit) <= 100 ? Number(limit) : 50;
-        const normalizedOffset = Number(offset) > 0 ? Number(offset) : 0;
-
-        const integrationEvents =
-            await this.integrationEventsService.getPaginatedEvents(
-                id,
-                normalizedLimit,
-                normalizedOffset,
-            );
-
         this.openApiService.respondWithValidation(
             200,
             res,
             integrationEventsSchema.$id,
-            {
-                integrationEvents: serializeDates(integrationEvents),
-            },
+            {},
```

Comment:
This looks a little weird, we're keeping the feature

Reply:
Same thing as the other one, I guess. It just looks for `integrationEvents`, thinks "oh, I can delete that" and then also cleans up where it's used. We should see if we can fix that somehow.

Reply:
See the comment above!